ScoutSuite/providers/gcp/services/cloudstorage.py

```diff
@@ -54,15 +53,11 @@ def parse_buckets(self, bucket, params):
 
 def get_cloudstorage_bucket_logging(bucket, bucket_dict):
     try:
-        logging = bucket.get_logging()
-        if logging:
-            bucket_dict['logging_status'] = 'Enabled'
-        else:
-            bucket_dict['logging_status'] = 'Disabled'
+        bucket_dict['logging_enabled'] = bucket.get_logging() != None
```

Comment:
Do we know for sure what the return type for `bucket.get_logging()` is? If it is anything else than a boolean, then the behaviour would change, as Python can treat various types as booleans implicitly. See the [following link](https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/boolean.html) for more info.

Using `bucket_dict['logging_enabled'] = bool(bucket.get_logging())` instead would preserve the current behaviour.

Reply:
`get_logging` returns either a dictionary or `None`. See [this link](https://googleapis.github.io/google-cloud-python/latest/storage/buckets.html#google.cloud.storage.bucket.Bucket.get_logging).

Reply:
The behavior is different but the outcome is still correct, according to the API. It specifically says `a dict with keys or None`.

As for the `!=`, I suggest we use GitHub's online file editing tool instead of leaving comments for small style-related issues, excepted if it's a recurring error.

Reply:
Oh, I didn't see the "w/ keys" part! Also, agreed for the syntax fix!